ouroboros-consensus/test-storage/Test/Ouroboros/Storage/FS.hs

```diff
@@ -83,71 +31,6 @@ tests tmpDir = testGroup "HasFS"
  The tests proper
 -------------------------------------------------------------------------------}
 
-{-
-test_touchFile :: Assertion
-test_touchFile = do
-    FS.touch ["foo.txt"] (Folder mempty) @?=
-        (Folder $ M.fromList [("foo.txt", File BS.empty)])
-    let action = FS.touch ["demo", "bar.md"] . FS.touch ["demo", "foo.txt"]
-    action (Folder $ M.fromList [("demo", Folder mempty)]) @?=
-        (Folder $ M.fromList [
-            ("demo", Folder $ M.fromList [
-              ("foo.txt", File BS.empty)
-            , ("bar.md", File BS.empty)
-            ]
-            )])
--}
-
-test_createDirectory :: Assertion
-test_createDirectory = do
-    withMockFS tryAny (\(Right (_, fs1)) -> Mock.mockFiles fs1 @?= result1) $ \HasFS{..} _err ->
-      createDirectory ["foo"]
-    withMockFS tryAny (\(Right (_, fs2)) -> Mock.mockFiles fs2 @?= result2) $ \HasFS{..} _err ->
-      createDirectoryIfMissing True ["demo", "foo"]
-  where
-     result1 = Folder $ M.fromList [("foo", Folder mempty)]
-     result2 = (Folder $ M.fromList [
-                 ("demo", Folder $ M.fromList [("foo", Folder mempty)]
-               )])
-
-test_listDirectory :: Assertion
```

Comment:
obsolete from TagCreateDirWithParentsThenListDir and TagCreateDirWithParentsThenListDirNotNull